ci/jenkins/Test.groovy.j2

```diff
@@ -199,6 +234,9 @@ stage('Test') {
     {{ method_name }}()
   },
   {% endfor %}
+  'unittest: CPU MINIMAL': {
```

Comment:
can you use the `m.test_step` macro here instead? That way we don't need to duplicate all the meta-logic (timeout, workspace, etc) just for this test

Reply:
I'm not sure how to do that. The `test_step` inlines the body while in this case, I am calling `run_unittest_minimal`. I could try to inline the method and use the macro, but that might not work due to the 64KB bytecode size/method limit in the JVM.

I also guess I could modify the macros.

Reply:
Ok, I am adding a new macro

Reply:
I meant something like https://github.com/apache/tvm/blob/main/ci/jenkins/Test.groovy.j2#L202-L218, if you do that it will automatically call the test method

Reply:
Yes, but I needed a different macro that did not include the step name (eg `unittest: CPU MINIMAL`, `frontend: aarch64 2 of 2`, `unittest: CPU`). 

https://github.com/apache/tvm/pull/12178/files#diff-1a31fe4681520821e63f0bec0a963c86b8eea7af2c7130b28eba0b987a5fc21bR86